fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/RepositoryObjectLoader.java

```diff
@@ -143,5 +143,4 @@ public RepositoryObject getRepositoryObject(PID pid) {
             }
         }
     }
-
```

Comment:
Did you want to remove this?

Reply:
I had added a new method here and then removed it a commit or two later. I've undone this change now